packages/office-ui-fabric-react/src/components/Toggle/Toggle.base.tsx

```diff
@@ -89,10 +88,11 @@ export class ToggleBase extends BaseComponent<IToggleProps, IToggleState> implem
                 disabled={disabled}
                 id={this._id}
                 type="button"
+                role="switch" // ARIA 1.1 definition; "checkbox" in ARIA 1.0
                 ref={this._toggleButton}
                 aria-disabled={disabled}
-                aria-pressed={checked}
-                aria-label={ariaLabel ? ariaLabel : label}
+                aria-checked={checked}
+                aria-label={ariaLabel ? ariaLabel : onAriaLabel}
```

Comment:
wouldn't this be breaking for people that use label, or places that have passed in 'click to turn off xyz' for the onAriaLabel?

Reply:
Agree that this is a breaking change. It is very unfortunate but maintaining backwards compat means we need to support what the old prop used to do - if users are using.

Reply:
Could we just keep the old behavior, mark it as @deprecated (so that people using depecations tslint get fail), and use `warnDeprecations(['onAriaLabel', etc])` to generate a console warning?

Reply:
No, it is not breaking for `label` users. `label` is already built to render as <label for="_id">, and this is used in the calculation by user agents. Part of this code change is also the removal of this bad redundancy. In Accessibility, it is bad form to supply multiple values for calculation that are the same, and even worse to supply multiple values for calculation that are different.

This code change also removed <label for="_id"> from the stateText. It is highly broken to have multiple labels for the same _id.

Reply:
Fixed fully